src/tests/ftest/SConscript

```diff
@@ -10,6 +10,8 @@ def scons():
     env.Install(ftest_install_dir, Glob('*.*'))
 
     dirs = ['aggregation', 'checksum', 'container', 'control',
+            'daos_perf', 'daos_racer', 'daos_vol',
+            'ior', 'mdtest',
```

Comment:
The io folder was re-org per Johann's request.
I'm on the fence for this one. While I do not disagree with you, I also think it is important and deserving for these tools/plugins to have their own directory to better scope/organize the tests, rather than having a big io dumping ground.
I think what is more important is to have accurate and effective tags on the tests so they can used more effectively in testing new features or feature changes. Changing tags is out of scope in this PR and will be discussed in the near future with inputs from developers/test engineers